pip-compile.sh

```diff
@@ -20,6 +20,7 @@ source_venv() {
 SCRIPT_DIR=$(dirname "$0")
 
 source_venv piptools_venv
-pip install -r "${SCRIPT_DIR}"/piptools_requirements3.txt
+pip install -r "${SCRIPT_DIR}"/piptools_requirements.txt
 pip install pip-tools
```

Comment:
what's the general opinion on pinning vs. not pinning `pip-tools`? it may spuriously break developer flow (if there's a backwards-compat change?) but i think that's fine

Reply:
I think it's alright to float since its output is checked into the repo